content/doc/book/pipeline/pipeline-best-practices.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: section
+title: Pipeline Best Practices
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author: Alex Taylor
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+:toc:
+endif::[]
+
+= Applying best practices to Pipelines
+
+== General
+
+=== Using Groovy code in Pipelines as glue
+
+Make sure to use Groovy code as a way of connecting together a set of actions rather than the main functionality. 
```

Comment:
Use Groovy code to connect a set of actions rather than as the main functionality of your Pipeline.